saritasa_invocations/_config.py

```diff
@@ -113,11 +124,103 @@ class AlembicSettings:
     )
 
 
+class DBSettings:
+    """Settings for db module."""
+
+    password_pattern: str = "Password: "
+    load_dump_command: str = (
+        "psql "
+        "{additional_params} "
+        "--dbname={dbname} "
+        "--host={host} "
+        "--port={port} "
+        "--username={username} "
+        "--file={file}.sql"
+    )
+    dump_filename: str = "local_db_dump"
+    load_additional_params: str = "--quite"
+    dump_command: str = (
+        "pg_dump "
+        "{additional_params} "
+        "--dbname={dbname} "
+        "--host={host} "
+        "--port={port} "
+        "--username={username} "
+        "--file={file}.sql"
+    )
+    dump_additional_params: str = "--no-owner"
+
+
+# This mapping should not be filled manually. You just need create an instance
+# of `K8SDataBaseSettings` and it will be auto added to this mapping
```

Comment:
I do not see any other mentions of `K8SDataBaseSettings`, maybe you meant `K8SSettings`?